app/user/rest.py

```diff
@@ -629,7 +629,18 @@ def get_all_users():
 @user_blueprint.route("/report-all-users", methods=["GET"])
 def report_all_users():
     users = dao_report_users()
-    return jsonify(data=users.serialize()), 200
+    download_users = []
+    for user in users:
+        new_user = {
+            "name": user[0],
+            "email_address": user[1],
+            "mobile_number": user[2],
+            "service": user[3],
+        }
+        download_users.append(new_user)
+    print(f"download_users {download_users}")
+    dump = json.dumps(download_users)
```

Comment:
Seems like this isn't necessary.

jsonify() is used to replace json.dumps(), I don't see the point of dumps first, seems error-prone.

https://www.geeksforgeeks.org/use-jsonify-instead-of-json-dumps-in-flask/

Reply:
I added the json.dumps() for a reason ...

Reply:
It looks like a further adjustment was made here - @xlorepdarkhelm and @terrazoon, is everything in good shape now?  If yes, @xlorepdarkhelm could you please review again?  Thanks!

Reply:
It's fine as far as I'm concerned.